packages/dapper/src/typeguards.ts

```diff
@@ -0,0 +1,23 @@
+export const isNumberArray = (value: unknown): value is number[] => {
```

Comment:
Ironically, this will actually pass for `[NaN]`, since `typeof NaN` is also `"number"`. If intended, that's fine, just making sure.

Reply:
Added check for NaN and some simple test cases for type guards.